collection-scripts/gather_apiserver_watchlog

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+WATCH_LOG_PATH=${OUT:-"${BASE_COLLECTION_PATH}/node_watch_log"}
+
+mkdir -p "${WATCH_LOG_PATH}"/
+
+function get_dump_off_node {
+    local debugPod=""
+    
+    #Get debug pod's name
+    debugPod=$(oc debug --to-namespace="default" node/"$1" -o jsonpath='{.metadata.name}')
+
+    #Start Debug pod force it to stay up until removed in "default" namespace
+    oc debug --to-namespace="default" node/"$1" -- /bin/bash -c 'sleep 300' > /dev/null 2>&1 &
+
+    #Mimic a normal oc call, i.e pause between two successive calls to allow pod to register
+    sleep 2
+    oc wait -n "default" --for=condition=Ready pod/"$debugPod" --timeout=30s
+
+    if [ -z "$debugPod" ]
+    then
+      echo "Debug pod for node ""$1"" never activated"
+    else
+      #Copy API watch log
+      echo "Copying core dumps on node ""$1"""
+      oc cp  --loglevel 1 -n "default" "$debugPod":/host/var/log/kube-apiserver/termination.log "${WATCH_LOG_PATH}"/"$1"_termination.log > /dev/null 2>&1 && PIDS+=($!)
```

Comment:
If the file name is called "termination.log", the script and the function name output should be called  "kube-apiserver termination logs".  Watchlog doesn't make sense

Reply:
Updated. Thanks